Boards/CYD-E32R32P/Source/Configuration.cpp

```diff
@@ -0,0 +1,76 @@
+#include "devices/SdCard.h"
+#include "devices/Display.h"
+
+#include <Tactility/hal/Configuration.h>
+#include <Tactility/lvgl/LvglSync.h>
+#include <PwmBacklight.h>
+
+#define CYD_SPI_TRANSFER_SIZE_LIMIT (240 * 320 / 4 * 2)
```

Comment:
This one can be moved to `Display.h` and changed to this:
`constexpr auto LCD_SPI_TRANSFER_SIZE_LIMIT = LCD_BUFFER_SIZE * LV_COLOR_DEPTH / 8;`